README.md

```diff
@@ -268,6 +268,21 @@ $ remme node-account transfer-tokens-from-frozen-to-unfrozen
 }
 ```
 
+Transfer tokens from unfrozen reputational balance to operational balance (executable only on the machine which runs the node) — `remme node-account transfer-tokens-from-unfrozen-to-operational`.
+
+| Arguments | Type    | Required | Description          |
+| :------:  | :-----: | :------: | -------------------- |
+| amount    | Integer | Yes      | Amount to transfer.  |
+
+```bash
+$ remme node-account transfer-tokens-from-unfrozen-to-operational --amount=1000
+{
+    "result": {
+        "batch_id": "045c2b7c43a7ca7c3dc60e92714c03265572a726d1fae631c39a404eaf97770e3f6a7a8c35c86f6361afb2e4f12b4a17d71a66a19158b62f30531ab32b62f06f"
```

Comment:
```suggestion
        "batch_identifier": "045c2b7c43a7ca7c3dc60e92714c03265572a726d1fae631c39a404eaf97770e3f6a7a8c35c86f6361afb2e4f12b4a17d71a66a19158b62f30531ab32b62f06f"
```

Reply:
"batch_id" field is returned by Remme core. If I change it in documentation it will not correspond to the reality.

Reply:
Where is the change?